kwave/utils/mapgen.py

```diff
@@ -182,7 +181,6 @@ def make_cart_bowl(bowl_pos: np.ndarray, radius: float, diameter: float, focus_p
         # return bowl
         raise NotImplemented("make_cart_disc")
 
-    # compute arc angle from chord (ref: https://en.wikipedia.org/wiki/Chord_(geometry))
```

Comment:
Why did you delete the comment line? Isn't it useful?

Reply:
you're right!